Automattic/wp-calypso
# Update copy of Security Real-time upgrade nudge

### Changes proposed in this Pull Request

Updated the copy of the upgrade nudge for Security Real-time to match the copydeck

### Testing instructions

- Select a self-hosted Jetpack site with Security Daily purchased
- Visit the _Plans_ section with the offer reset flow enabled
- See the nudge in the features section of Security Daily and check its copy


### Screenshots

Updated nudge
<img width="470" alt="Screen Shot 2020-08-20 at 5 04 38 PM" src="https://user-images.githubusercontent.com/1620183/90826190-074dbc80-e308-11ea-86b1-0028ae190d72.png">
